### ISSUE TITLE
TabIndex on Inventory Item form

### ISSUE THREAD

**Issue by Dilden** [Dilden/helth#254]
When tabbing through Items in the Inventory, the "Delete" button is shown at the end but third in the tab order. It should be last. 

The duplicate + edit buttons aren't rendered in the Item.svelte and are instead shown in the List.svelte, which is why the delete is in the tab order before. 

It could all also be done in a flex row instead of a div floating an element to the right.
